mantidproject/mantid
# Move Linked Wiki Files to Main Docs Directories

**Description of work.**
Moves a number of documentation files off of the wiki and into our internal `.rst` structure. This also helps to remove the number of `www.` links in the project, which we are aiming to shrink. 

This could be split into multiple PRs if specialists for the specific parts that these docs cover want to review them separately. This got a little bigger than I expected. 

**To test:**
1. Delete doc-trees
2. Build `docs-html` and/or `docs-qthelp`
3. Check each of the new pages and check all the links work
4. Check that changed links to these new docs go to the right files and work within he QtHelp window in workbench without opening web pages (the link in the dev-docs is an exception to this, but that won't work anyway until merged). 


Partial work on #31340

*This does not require release notes* because **is is only moving docs to our git trackable system. **



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
